docs/TRANSCRIPTION.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Build a Real-Time Transcription Backend with Astro
+description: Learn how to build a backend to initiate Real Time Transcription using Astro.
+---
+
+50% of Americans watch content with subtitles. Why can't we have this with video calls? Actually, we can do this by using Real-Time Transcription with Agora.
+
+This guide is the third part in a series about building backends with Astro. The first guide covered Token Generation with Astro so we can secure our video calls. We will use the token generation logic in this guide. The second guide was about recording your video to a storage provider of your choice. Real-Time Transcription is a separate feature, so it does not rely on the cloud recording guide, but we will need to generate tokens. Let's get started.
```

Comment:
Do you think it's worth linking to the previous guides? I the event that a user found this one before the others.

Reply:
Technically, this statement isn't completely accurate:
> Real-Time Transcription is a separate feature, so it does not rely on the cloud recording guide, but we will need to generate tokens. Let's get started.

While RTT is a separate product, there is some overlap with cloud recording when storing the transcription data especially transcriptions stored along-side recordings.

Reply:
The tokens guide is linked, but you are right. I will link to the cloud recording guide as well.

As far as the other point, I didn't say it doesn't rely on cloud recording *the feature*, just that it doesn't rely on *the guide* that was previously written in this series

Reply:
Ok re-reading that again I think I initially misunderstood the statement.